OcapReplaySaver2/OcapReplaySaver2.cpp

```diff
@@ -144,6 +145,7 @@ namespace {
         int httpRequestTimeout{ 120 };
         int traceLog{ 0 };
         std::string logsDir{ "./OCAPLOG" };
+        std::string tempDir{ "./OCAPTMP" };
```

Comment:
@indig0fox with that change it will use this folder by default.
Should i change it back to the old behaviour, so that i will use the default os temp folder until you configure a custom one?

Reply:
I think keeping existing log behavior and default folder, and adding a default temp dir alongside it in the Arma root for storing backup .gz, is the most intuitive way forward. I would remove the os default temp folder usage entirely. With this, logs will always tie to recordings in the a3 root regardless of profile, user account, or install environment.